vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -0,0 +1,46 @@
+#page-container .ag-theme-alpine.vizro {
+  --ag-background-color: var(--main-container-bg-color);
+  --ag-foreground-color: var(--text-primary);
+  --ag-odd-row-background-color: var(--main-container-bg-color);
+  --ag-header-foreground-color: var(--text-primary);
+  --ag-header-background-color: var(--main-container-bg-color);
+  --ag-header-column-resize-handle-display: none;
+  --ag-font-size: 14px;
+  --ag-borders: none;
+  --ag-icon-font-family: agGridBalham;
+  --ag-icon-size: 14px;
+}
+
+#page-container .ag-theme-alpine .ag-header-cell {
+  border-bottom: 1px solid var(--border-subtle-alpha-02);
+  padding-left: 0px;
```

Comment:
I wouldn't remove the padding from only one side if that's possible. Otherwise, it will be difficult to center content properly. 

A few things I've noticed that I think we can definitely fix:
- set header height to 40px
- set row height to 48px
- set font-size everywhere to 14px (currently I see 15px and 14px mixed)
- fix padding in the cells

Would be great if you could double-check the Figma specifications on dev mode. Let me know if we should connect on that one, but e.g. for the cells we have this:

```
display: flex;
height: 48px;
padding: 0px 12px;
align-items: center;
align-self: stretch;
```